wazuhapp/Docker/build.sh

```diff
@@ -34,12 +34,18 @@ build_package(){
     yarn
     yarn build
 
-    find /source/build/ -name wazuh* -exec cp {} /wazuh_app/${wazuh_app_pkg_name} \;
+    if [[ "${checksum}" == "yes" ]]; then
```

Comment:
Change this if block as said in the Debian `build.sh` script.

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/192/commits/9d09627160a93091dae44ea05cd9ca3429cd8f21